client/src/components/infoDrawer/infoFormat.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 
 const renderContributors = (contributors, affiliations, skeleton) => {
   // eslint-disable-next-line no-constant-condition --  Temp removed contributor section to avoid publishing PII
-  if (contributors?.length === 0 && true) return null;
+  if (!contributors || (contributors.length === 0 && true)) return null;
```

Comment:
Would you want to continue leaving the `?` in contributors.length? I get confused by undefined vs null but it seems like `!contributors` would catch the null case but not when contributors is undefined?

Reply:
`!contributors` will be checking for falsely values which includes undefined and null